src/components/Documents/DocumentTableRow.jsx

```diff
@@ -64,6 +65,11 @@ const DocumentTableRow = ({ document }) => {
           <FileOpenIcon />
         </IconButton>
       </StyledTableCell>
+      <StyledTableCell align="center">
+        <IconButton type="button" onClick={() => handleAclPermissionsModal('document', name)}>
+          <SettingsIcon />
```

Comment:
Just to test, I would like to try out other icons and see how we feel about them. For example, `LockOutlinedIcon` could be used for documents that have no permissions given, then it would switch to `LockOpenOutlinedIcon` whenever any sort of permission is granted. Or perhaps `KeyOffOutlinedIcon` and `KeyOutlinedIcon` instead.

My thinking is it might provide the user with a bit more information at a glance.

Haven't tested these myself yet but I'm curious.

Reply:
I like the idea of a lock or key icon for permissions rather than a gear icon. Gear icon is a bit vague.

Reply:
Or maybe even a sharing icon:
https://mui.com/material-ui/material-icons/?query=share&selected=Share

Reply:
I don't know how to get the info of whether a document has been shared or not. Maybe we could try that in another PR? I changed the gear to a Share Icon though.

Reply:
I think determining whether or not it's been shared can be left for a future PR. We could even add a simple `useState` that toggles the icon when clicked, like with the pin in `ContactListTableRow.jsx`. Assuming we want to go that route.